Resources/Prototypes/game_presets.yml

```diff
@@ -109,3 +109,16 @@
   rules:
     - Pirates
     - BasicStationEventScheduler
+
+- type: gamePreset
+  id: Revolution
+  alias:
+    - Revo
+    - Revolutionary
+    - Revolutionaries
+  name: revolution-title
+  description: revolution-description
+  showInVote: false
+  rules:
+    - Revolution
+    - BasicStationEventScheduler
```

Comment:
It shouldn't be on the scheduler if it's not supposed to be used outside of roundstart.